src/hotspot/cpu/x86/macroAssembler_x86_pow.cpp

```diff
@@ -1,5 +1,5 @@
 /*
-* Copyright (c) 2016, Intel Corporation.
+* Copyright (c) 2016, 2021, Intel Corporation.
```

Comment:
Please, don't modify copyright line of other company. Add new line for your company.